src/echo-app/src/main/index.ts

```diff
@@ -60,6 +63,14 @@ app.whenReady().then(() => {
 
   createWindow()
 
+  if (is.dev) {
+    function notifyOfRemoteAuth(token: string) {
+      mainWindow?.webContents.send('AUTH_TOKEN_RECEIVED', { TOKEN_RECEIVED: token })
```

Comment:
We have context isolation/sandbox turned off so you shouldn't need IPC to send the token.

Reply:
What's the best way to do this without ipc?